app/src/services/reportFile.service.js

```diff
@@ -107,20 +107,22 @@ class FileService {
           question = { name: response.name, label: { [language]: response.name } };
           questions[record.attributes.report].push(question);
         }
-        // check if the answer is an image
+        // check if the answer is a file
         if (response.value && response.value.startsWith("https://s3.amazonaws.com")) {
```

Comment:
Pin for the future, this file check should be based on the question type instead of the the response starting with the url string as this will consider even text responses that begin with this as a file to be downloaded